app/models/form526_submission.rb

```diff
@@ -68,6 +60,36 @@ def start
     start_evss_submission_job
   end
 
+  def start_rrd_job(rrd_processor_class, backup_processor_class = nil)
+    workflow_batch = Sidekiq::Batch.new
+    workflow_batch.on(
+      :success,
+      'Form526Submission#rrd_complete_handler',
+      'submission_id' => id
+    )
+    if backup_processor_class
+      workflow_batch.on(
+        :death,
+        'Form526Submission#rrd_processor_failed_handler',
+        'submission_id' => id
+      )
+    end
```

Comment:
If a `backup_processor_class` is provided and the job for the `rrd_processor_class` fails, then call `rrd_processor_failed_handler`, which will restart the RRD job except using the `backup_processor_class`.

Reply:
Does this override, or supplement, the `sidekiq_retries_exhausted` behavior declared on the class itself?

Since Form526BaseJob has that block copied over from DisabilityCompensationJob, I'm worried that if something goes wrong, it'll trigger both the baked-in EVSS submission, and the backup processor here.

Reply:
It supplements `sidekiq_retries_exhausted`.

From https://github.com/mperham/sidekiq/wiki/Batches#callbacks:
> If a job fails all retries and dies, it will fire any :death callbacks.

Reply:
Pushed a commit that removes `sidekiq_retries_exhausted` in the old and new refactored jobs, and updates the `:death` callback to handle exhaused retries using Sidekiq::Batch's approach.